teal/notifier/status.tl

```diff
@@ -54,6 +54,9 @@ function StatusModule._create_win()
       col = vim.o.columns,
       zindex = 10,
     })
+    local winhlns = api.nvim_create_namespace("notifier_winhl")
+    api.nvim_win_set_hl_ns(StatusModule.win_nr, winhlns)
+    api.nvim_set_hl(winhlns, "NormalFloat", { bg = "None" })
```

Comment:
We should expose a new highlight group here.

Reply:
Really? This is just for the background of the message which I think everyone would want to be same as `Normal` background

Reply:
Having it configurable but with a sane default is pretty much equivalent.
I'd like to have it configurable.

Reply:
I can't think of a good way to do this, making a highlight group called `NotifierFloat` with `bg = "None"` and linking the floatwin's `NormalFloat` to `NotifierFloat` doesn't work

Reply:
I'll look into it then.
But it should work just like any other highlight group.
Maybe `bg=None` should be added to other highlight groups too.

Reply:
It is different from other highlight groups - putting `bg = None` in other `NotifierXX` groups will use `NormalFloat`'s bg, just like putting `bg=None` in a normal (non-floating) window's highlight group will use `Normal`'s bg

Reply:
'kay, then I'll check that.
Do you mind if I fix the issue on my end ?